test/soak/cauldron/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: cauldron
+
+TAG = 1.0
+
+cauldron:
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build -a -installsuffix cgo --ldflags '-w' cauldron.go
+
+container:	cauldron
+	docker build -t kubernetes/cauldron:$(TAG) .
```

Comment:
Yes, but I can wait until I am ready for that? I expect to rapidly iterate this code until it gets to a suitable point for a "1.0".